src/main/java/com/example/Main.java

```diff
@@ -1,7 +1,21 @@
 package com.example;
 
+/**
+ * Main class that contains void main
+ *
+ * @author Elena Yevsyukova
+ * @version 1.1
+ */
 public class Main {
+    /**
+     * Here starts point of the program
+     *
+     * @param args command line values
+     */
     public static void main(String[] args) {
-
+        /**
+         * Object of the Customer class that takes a String
+         */
```

Comment:
You don't add Javadoc for variables. Instead, you name variables so that it's clear what's inside them.